app/views/homepage/_references.html.erb

```diff
@@ -0,0 +1,99 @@
+<%
+  references = [
+    {
+      title: "Pipeline",
+      links: [
+        {
+          title: "Step types",
+          link: "/docs/pipelines/step-reference"
+        },
+        {
+          title: "Conditionals",
+          link: "/docs/pipelines/conditionals"
+        },
+        {
+          title: "Environment variables",
+          link: "/docs/pipelines/environment-variables"
+        },
+        {
+          title: "Emojis",
+          link: "https://github.com/buildkite/emojis#emoji-reference",
+          is_external: true
+        }
+      ]
+    },
+    {
+      title: "Agent",
+      links: [
+        {
+          title: "Configuration settings",
+          link: "/docs/agent/v3/configuration"
+        },
+        {
+          title: "Lifecycle hooks",
+          link: "/docs/agent/v3/hooks"
+        },
+        {
+          title: "CLI commands",
+          link: "/docs/agent/v3/cli-reference"
+        },
+        {
+          title: "Elastic Stack template parameters",
+          link: "/docs/agent/v3/elastic-ci-aws/parameters"
+        }
+      ]
+    },
+    {
+      title: "API",
+      links: [
+        {
+          title: "REST API",
+          link: "/docs/apis/rest-api"
+        },
+        {
+          title: "GraphQL API",
+          link: "/docs/apis/graphql-api"
+        },
+        {
+          title: "Webhooks",
+          link: "/docs/apis/webhooks"
+        }
+      ]
+    },
+    {
+      title: "Test Analytics",
+      links: [
+        {
+          title: "JSON test results",
+          link: "/docs/test-analytics/importing-json"
+        },
+        {
+          title: "Environment variables",
+          link: "/docs/test-analytics/ci-environments"
+        }
+      ]
+    }
+  ]
+%>
+
+<section class="References">
+  <div class="container container-narrow">
+    <h2 class="HomePage__section-heading">APIs and references</h2>
```

Comment:
I'm not loving the mixed levels of BEM and surgical classes as it makes the styles hard to follow.

Since we've got SASS mixins at our disposal could you do something like the following?

```scss
.References { 
  @import container("narrow")
}

.References__Header {
  @include section-header;
}
```

Maybe the surgical classes (`container`) is OK but definitely having `Homepage__section-heading` nested under `References` is what I would consider to be a BEM violation

Reply:
Personally, the developer experience of using BEM in Docs isn't helpful with styling and communicating the component structure. It's a pattern I've inherited and I'd be keen to move away from this. We should chat about other ways to work with the frontend.

Reply:
I've pushed up a commit to use the Sass mixin fa627346668f78e25a8ecc26a35790b867372854